llmnrd.c

```diff
@@ -257,7 +270,11 @@ int main(int argc, char **argv)
 		rm_pid_file = true;
 	}
 
-	log_info("Starting llmnrd on port %u, hostname %s\n", port, hostname);
+	log_info("Starting llmnrd on port %u, hostname(s): ", port);
+    for(name_i = 0; name_i < name_count; ++name_i)
+        log_info("%s ", hostnames[name_i]);
+    log_info("\n");
```

Comment:
Please use tabs to indent. Also, I think continuing the log message won't work when using syslog. Maybe construct the hostname string first and then log it?